nipype/interfaces/fsl/tests/test_BEDPOSTX.py

```diff
@@ -1,3 +1,6 @@
 # AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
 from nipype.testing import assert_equal
 from nipype.interfaces.fsl.dti import BEDPOSTX
+
+#NOTE_dj: this test has only import statements
+#NOTE_dj: this is not a AUTO test!
```

Comment:
I've restored the file, but change the comment. However, I did not notice any new test_auto_BEDPOSTX .